### PR TITLE
PMM-3659 Add postgres_exporter to PMM-agent.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: api/Gopkg.lock [hunk 1]
```diff
@@ -524,30 +519,14 @@
...
-    "github.com/grpc-ecosystem/grpc-gateway/runtime",
-    "github.com/grpc-ecosystem/grpc-gateway/utilities",
-    "github.com/mwitkow/go-proto-validators",
     "github.com/mwitkow/go-proto-validators/protoc-gen-govalidators",
-    "google.golang.org/genproto/googleapis/api/annotations",
     "google.golang.org/grpc",
-    "google.golang.org/grpc/codes",
-    "google.golang.org/grpc/grpclog",
     "google.golang.org/grpc/metadata",
-    "google.golang.org/grpc/status",
```
Comment (AlekSi) [inline api/Gopkg.lock]: All those removals look wrong… Did you use `make init` before `make gen`?

Comment (AlekSi) [reply api/Gopkg.lock]: Perfect timing! :D

Comment (BupycHuk) [reply api/Gopkg.lock]: yes, I've already reverted it.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-317/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-317/

**Pull request merged**